test/integration/framework/etcd.go

```diff
@@ -84,7 +83,15 @@ func startEtcd(output io.Writer) (func(), error) {
 	return stop, nil
 }
 
-var initGRPCOnce sync.Once
+func init() {
+	// Quiet etcd logs for integration tests
+	// Comment out to get verbose logs if desired.
+	// This has to be done before there are any goroutines
+	// active which use gRPC. During init is safe, albeit
+	// then also affects tests which don't use RunCustomEtcd
+	// (the place this was done before).
+	grpclog.SetLoggerV2(grpclog.NewLoggerV2(io.Discard, io.Discard, os.Stderr))
```

Comment:
should we care about the verbosity level too?

https://pkg.go.dev/google.golang.org/grpc/grpclog#LoggerV2

or just reduce the verbosity can be enough?